service/lib/agama/dbus/software/manager.rb

```diff
@@ -123,7 +123,7 @@ def initialize(backend, logger)
 
         def available_base_products
           backend.products.map do |id, data|
-            [id, data["name"], { "description" => data["description"] }].freeze
+            [id, data["name"], { "description" => localized_description(data) }].freeze
```

Comment:
my wild guess here is that it won't raise signal when localization changes. So if UI cache list of base products, it will use it with old locale.